### ISSUE TITLE
Support field without a type hint

### ISSUE THREAD

**Issue by samuelcolvin** [koxudaxi/pydantic-pycharm-plugin#37]
(Sorry for so many issues, just noting things when I find them)

In the following example:
```py
from pydantic import BaseModel

class Foobar(BaseModel):
    x: int
    y = 10

Foobar(x=1, y=2)
```

pydantic knows that `y` is an `int`. (This will be discouraged in v1 because of inferring field order, but will definitely still be possible)

The plugin is currently ignoring `y` altogether:

![image](https://user-images.githubusercontent.com/4039449/63106866-b3352d00-bf7b-11e9-820e-23b8985898a7.png)

**Comment by koxudaxi** [comment]
Thank you for creating issues
No problem.  I want feedback 🚀 

First, the behavior is wrong. The plugin can't detect value without type-hint. I will fix it.

> pydantic knows that y is an int. 

Second, I will check that can do `Type inference` in the plugin

**Comment by koxudaxi** [comment]
I fix the bugs.
If a filed doesn't have type-hint then, PyCharm detects type from default-value.
<img width="440" alt="ss2019-08-16 22 41 55" src="https://user-images.githubusercontent.com/630670/63173076-10170d00-c07a-11e9-8d58-4648978d44d9.png">

**Comment by koxudaxi** [comment]
Fixed version 0.0.13 is released. It's available in the marketplace.
I close the issue.

**Comment by koxudaxi** [comment]
Thank you very much.
